src/functionalTest/resources/features/F-007/S-071.td.json

```diff
@@ -4,191 +4,777 @@
+          }
+        }, {
+          "roleType" : {
+            "mandatory" : true,
+            "values" : [ "CASE" ]
+          },
+          "grantType" : {
+            "mandatory" : true,
+            "values" : [ "SPECIFIC", "CHALLENGED" ]
+          },
+          "classification" : {
+            "mandatory" : true,
+            "values" : [ "PUBLIC" ]
+          },
+          "beginTime" : {
+            "mandatory" : true
+          },
+          "attributes" : {
+            "caseId" : {
+              "mandatory" : true
+            }
+          }
+        } ]
+      }, {
+        "name" : "tribunal-caseworker",
+        "label" : "Tribunal Caseworker",
+        "description" : "Tribunal caseworker",
+        "category" : "LEGAL_OPERATIONS",
+        "patterns" : [ {
+          "roleType" : {
+            "mandatory" : true,
+            "values" : [ "ORGANISATION" ]
+          },
+          "grantType" : {
+            "mandatory" : true,
+            "values" : [ "STANDARD" ]
+          },
+          "classification" : {
+            "mandatory" : true,
+            "values" : [ "PUBLIC", "PRIVATE" ]
+          },
+          "attributes" : {
+            "jurisdiction" : {
+              "mandatory" : true,
+              "values" : [ "IA" ]
+            },
+            "primaryLocation" : {
+              "mandatory" : true
+            }
+          }
+        }, {
+          "roleType" : {
+            "mandatory" : true,
+            "values" : [ "CASE" ]
+          },
+          "grantType" : {
+            "mandatory" : true,
+            "values" : [ "SPECIFIC" ]
+          },
+          "classification" : {
+            "mandatory" : true
+          },
+          "attributes" : {
+            "jurisdiction" : {
+              "mandatory" : true
+            },
+            "caseType" : {
+              "mandatory" : true
+            },
+            "caseId" : {
+              "mandatory" : true
+            }
+          }
+        } ]
+      }, {
+        "name" : "senior-tribunal-caseworker",
+        "label" : "Senior Tribunal Caseworker",
+        "description" : "Senior Tribunal caseworker",
+        "category" : "LEGAL_OPERATIONS",
+        "patterns" : [ {
+          "roleType" : {
+            "mandatory" : true,
+            "values" : [ "ORGANISATION" ]
+          },
+          "grantType" : {
+            "mandatory" : true,
+            "values" : [ "STANDARD" ]
+          },
+          "classification" : {
+            "mandatory" : true,
+            "values" : [ "PUBLIC", "PRIVATE" ]
+          },
+          "attributes" : {
+            "jurisdiction" : {
+              "mandatory" : true,
+              "values" : [ "IA" ]
+            },
+            "primaryLocation" : {
+              "mandatory" : true
+            }
+          }
+        } ]
+      }, {
+        "name" : "petitioner's-solicitor",
+        "label" : "Petitioner's Solicitor",
+        "description" : " Petitioner's Solicitor",
```

Comment:
whitespace

Reply:
you mean white space in the description?

Reply:
yeah